src/SimpleEventStore/SimpleEventStore.CosmosDb.Tests/AzureCosmosDbEventStoreInitializing.cs

```diff
@@ -26,47 +25,20 @@ public async Task when_initializing_all_expected_resources_are_created()
             var collectionName = "AllExpectedResourcesAreCreated_" + Guid.NewGuid();
             var storageEngine = await InitialiseStorageEngine(collectionName, collectionThroughput: TestConstants.RequestUnits);
 
-            var database = (await client.ReadDatabaseAsync(databaseUri)).Resource;
-            var collection = (await client.ReadDocumentCollectionAsync(UriFactory.CreateDocumentCollectionUri(DatabaseName, collectionName))).Resource;
-
-            var storedProcedure = client.CreateStoredProcedureQuery(collection.SelfLink)
-                .Where(r => r.Id.StartsWith("appendToStream-"))
-                .AsEnumerable()
-                .OfType<StoredProcedure>()
-                .Single();
-
-            var offer = client.CreateOfferQuery()
-                .Where(r => r.ResourceLink == collection.SelfLink)
-                .AsEnumerable()
-                .OfType<OfferV2>()
-                .Single();
-
-            Assert.That(storedProcedure, Is.Not.Null);
-            Assert.That(offer.Content.OfferThroughput, Is.EqualTo(TestConstants.RequestUnits));
-            Assert.That(collection.DefaultTimeToLive, Is.Null);
-            Assert.That(collection.PartitionKey.Paths.Count, Is.EqualTo(1));
-            Assert.That(collection.PartitionKey.Paths.Single(), Is.EqualTo("/streamId"));
-            Assert.That(collection.IndexingPolicy.IncludedPaths.Count, Is.EqualTo(1));
-            Assert.That(collection.IndexingPolicy.IncludedPaths[0].Path, Is.EqualTo("/*"));
-            Assert.That(collection.IndexingPolicy.ExcludedPaths.Count, Is.EqualTo(3));
-            Assert.That(collection.IndexingPolicy.ExcludedPaths[0].Path, Is.EqualTo("/body/*"));
-            Assert.That(collection.IndexingPolicy.ExcludedPaths[1].Path, Is.EqualTo("/metadata/*"));
-        }
+            var database = client.GetDatabase(DatabaseName);
+            var collection = database.GetContainer(collectionName);
 
-        [Test]
```

Comment:
Not sure why we have removed this test?

Reply:
The V3 SDK doesn't seem to give us a way to view the default TLL 
![image](https://user-images.githubusercontent.com/30480171/72158958-4d769100-33b3-11ea-8ef6-bcee61aa849d.png)

Reply:
Re-added